fossology/fossology
# feat(test): Bypass API rate limit of github.

## Description

[Sometimes](https://travis-ci.org/fossology/fossology/jobs/421499631) the PHP tests failed because they hit the [API rate limit of GitHub](https://developer.github.com/v3/#rate-limiting).

### Changes

Replaced the API call in the tests with a test double.

## How to test

* Run the `RepositoryApiTest`.
* Check that the Aboutpage still works.
